boards/x86-multiboot-common/startup.c

```diff
@@ -52,7 +52,7 @@ extern const multiboot_header_t multiboot_header __attribute__((section("._multi
 const multiboot_header_t multiboot_header = {
     .magic = MULTIBOOT_HEADER_MAGIC,
     .flags = MULTIBOOT_HEADER_FLAGS,
-    .checksum = MULTIBOOT_HEADER_CHECKSUM,
+    .checksum = (-(MULTIBOOT_HEADER_MAGIC + MULTIBOOT_HEADER_FLAGS)),
```

Comment:
Actually, I wrote this intentionally as is to avoid similarities to the GPL version. (I know that legally this is rather useless.)